v3.1/Dockerfile

```diff
@@ -77,4 +87,8 @@ RUN apt-get update \
     && rm -r mapnik \
     && apt-get autoremove -y --purge $BUILD_DEPENDENCIES \
     && rm -rf /var/lib/apt/lists/* \
-    && ln -s /usr/local/lib/mapnik /usr/lib/mapnik
+    && ln -s /usr/local/lib/mapnik /usr/lib/mapnik \
+    && ln -s /usr/local/lib/mapnik /usr/local/lib/plugins 
```

Comment:
Why do you link the mapnik folder to plugin ?

Reply:
It is for mapnik-render to be able to access the plugins.